### PR TITLE
[release-2.5] Ability to disable alert forwarding

### PR DESCRIPTION
**Pull request by subbarao-meduri**
Signed-off-by: Subbarao Meduri <smeduri@redhat.com>

### CODE REVIEW

File: operators/endpointmetrics/pkg/rendering/renderer.go [hunk 1]
```diff
@@ -198,7 +198,7 @@ func Render(
 					s.GetName(),
 				)
 			}
-			// replace the hub alertmanager address
+			// replace the hub alertmanager address. Address will be set to null when alerts are disabled
```
Comment (marcolan018) [inline operators/endpointmetrics/pkg/rendering/renderer.go:201]: is this pr handle the *KS scenario?

Comment (subbarao-meduri) [reply operators/endpointmetrics/pkg/rendering/renderer.go:201]: Yes @marcolan018

Comment (subbarao-meduri) [reply operators/endpointmetrics/pkg/rendering/renderer.go:201]: When alerts are disabled, they are disabled for *all* clusters, not just for OCP. Are there specific concerns @marcolan018 ?

Comment (marcolan018) [reply operators/endpointmetrics/pkg/rendering/renderer.go:201]: I cannot find any related change related with *KS.
if alert forwarding disabled in *KS, just send an empty url to the prometheus?

**Pull request closed without merging**